examples/htool.c

```diff
@@ -433,6 +474,10 @@ static const struct htool_cmd CMDS[] = {
                 {HTOOL_FLAG_VALUE, 's', "start", "0", .desc = "start address"},
                 {HTOOL_FLAG_BOOL, 'v', "verify", "true"},
                 {HTOOL_POSITIONAL, .name = "source-file"},
+                {HTOOL_FLAG_VALUE, 'a', "address_mode", "3B/4B",
```

Comment:
Nit suggestion, can we move the FLAG_VALUE to before the POSITIONAL? Just for readability so the positional param does not get buried behind other flags.